### PR TITLE
Fixed wiki page generation

### PR DESCRIPTION
**Pull request by BrianFarnhill**

### CODE REVIEW

**Comment by BrianFarnhill** [comment]
@caadam This is literally the smallest code review you will ever get from me bloke :)

File: Tests/Generate-xSharePointWikiPages.ps1 [hunk 1]
```diff
@@ -9,7 +9,8 @@ $repoDir = Join-Path $PSScriptRoot "..\" -Resolve
 
 Get-ChildItem "$repoDir\modules\xSharePoint\**\*.schema.mof" -Recurse | `
     ForEach-Object { 
-        $result = (Get-MofSchemaObject $_.FullName) | Where-Object { $_.ClassName -eq $_.Name.Replace(".schema.mof", "") }
+        $mofFileObject = $_
+        $result = (Get-MofSchemaObject $_.FullName) | Where-Object { $_.ClassName -eq $mofFileObject.Name.Replace(".schema.mof", "") -and $_.FriendlyName -ne $null }
```
Comment (caadam) [inline Tests/Generate-xSharePointWikiPages.ps1]: This line isn't lone enough.

Comment (BrianFarnhill) [inline Tests/Generate-xSharePointWikiPages.ps1]: Sarcasm aside, I'm letting this one through. Thanks bud :)

**Comment by caadam** [comment]
Code review? More like line-of-code review!!! Approved!

**Pull request merged**